### ISSUE TITLE
Browser crashes when PiP video size is changed

### ISSUE THREAD

**Issue by choodique** [minbrowser/min#1229]
* **Min Version:** 1.16.0 
* **Operating system:** macOS 10.13.6

## Expected Behavior
Watching YouTube video in PiP mode and working in parallel in other tabs

## Actual Behavior
When i change the size of the YouTube video in PIP mode when another tab is opened, the browser crashes

## To Reproduce
* start browser
* open YouTube and play video
* click right button twice and select PiP mode
* open new tab and goto news.goole.com (example)
* try to resize PiP video
* browser crash

Can see video here - http://a.tmp.ninja/JcQJqH7UP8Cm.mov

**Comment by PalmerAL** [comment]
Duplicate of #1154.
